homeassistant/components/ecobee/climate.py

```diff
@@ -568,17 +571,47 @@ def extra_state_attributes(self) -> dict[str, Any] | None:
     @property
     def is_aux_heat(self) -> bool:
         """Return true if aux heater."""
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            "service_deprecation",
```

Comment:
I think this is stale. Use a more meaning full key, and do not forget to add it to `strings.json`

Reply:
Thanks. I've cleaned this up and included the strings.json updates.